lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -126,17 +126,13 @@
 
 .nx-dropdown-link {
   color: var(--nx-color-link);
-
-  &:hover {
-    color: var(--nx-color-link-hover);
-    text-decoration: underline;
-  }
+  text-decoration: underline;
 
   &.disabled {
     &, &:hover, &:focus {
       color: var(--nx-color-link-disabled);
       cursor: default;
-      text-decoration: none;
+      text-decoration: underline;
```

Comment:
Do we need this? I think it would inherit the text-decoration defined on line 129.

Reply:
> Do we need this? I think it would inherit the text-decoration defined on line 129.

Thanks! yeah we don't need it: 0937f12d715455aa795db5220d4c36c687140d46